terraform/modules/tf_lambda/iam.tf

```diff
@@ -17,27 +17,11 @@ resource "aws_iam_role" "role" {
   assume_role_policy = "${data.aws_iam_policy_document.lambda_execution_policy.json}"
 }
 
-// Base permissions - Allow creating logs and publishing metrics
-data "aws_iam_policy_document" "logs_metrics_policy" {
-  statement {
-    effect = "Allow"
-
-    actions = [
-      "cloudwatch:PutMetricData",
-      "logs:CreateLogGroup",
-      "logs:CreateLogStream",
-      "logs:PutLogEvents",
-    ]
-
-    resources = ["*"]
-  }
-}
-
-resource "aws_iam_role_policy" "logs_metrics_policy" {
-  count  = "${var.enabled}"
-  name   = "LogsAndMetrics"
-  role   = "${aws_iam_role.role.id}"
-  policy = "${data.aws_iam_policy_document.logs_metrics_policy.json}"
+// Attach write permissions for CloudWatch logs
+resource "aws_iam_role_policy_attachment" "logs_metrics_policy" {
+  count      = "${var.enabled}"
+  role       = "${aws_iam_role.role.id}"
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
```

Comment:
For reference: `AWSLambdaBasicExecutionRole – Grants permissions only for the Amazon CloudWatch Logs actions to write logs. You can use this policy if your Lambda function does not access any other AWS resources except writing logs.`

Reply:
It might be worth it to add this as a comment ^

Reply:
We are attaching this policy in addition to other policies (that comment refers to choosing the managed policy when creating the Lambda role from the AWS console). So every Lambda function will have this basic policy attached.